.travis.yml

```diff
@@ -4,6 +4,7 @@ dist: trusty
 
 jdk:
 - oraclejdk8
+- oraclejdk7
```

Comment:
Thanks @junron! This PR will be very useful! Please, it's not necessary to change this file, because Travis CI is only used for creating releases now.